.github/workflows/ruby.yml

```diff
@@ -0,0 +1,19 @@
+name: Tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 3.0
```

Comment:
Do you think it's worth using the github actions matrix to test both 2.7 and 3.0, like in the ci.yml file you removed?

Reply:
Done